sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -1329,15 +1329,15 @@ Command *Scheduler::GraphBuilder::connectDepEvent(
   ExecCGCommand *ConnectCmd = nullptr;
 
   try {
-    std::unique_ptr<detail::HostTask> HT(new detail::HostTask);
+    std::shared_ptr<detail::HostTask> HT(new detail::HostTask);
```

Comment:
Is this change NFC?

Reply:
Functionally, we co not change the intended behavior of the pointer. It does change the way it could be used in the future, though I do not see how that would be a problem. The benefit for the runtime is that the headers can have it be an incomplete type.